bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -412,9 +412,47 @@ def _canJsDisplay(self):
 
 
     def _getJsCode(self):
+        # Add extra primitives to canvas with custom colors, palette, gStyle
+        prim = self.drawableObject.GetListOfPrimitives()
+
+        style = ROOT.gStyle
+        colors = ROOT.gROOT.GetListOfColors()
+        palette = 0
+
+        # always provide gStyle object
+        if prim.FindObject(style):
+           style = 0
+        else:
+           prim.Add(style)
+
+        cnt = 0
+        for n in range(colors.GetLast()+1):
```

Comment:
I guess here you want `n` to have values from `0` to `colors.GetLast()`, right?

Reply:
Yes, range [0 .. colors.GetLast()]